spec/migrations/20160115111829_chargeback_rate_detail_currency_not_nil_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "spec_helper"
+require_migration
+
+describe ChargebackRateDetailCurrencyNotNil do
+  let(:chargeback_rate_detail_stub) { migration_stub(:ChargebackRateDetail) }
+
+  migration_context :up do
+    it "change to default currency" do
+      # if the chargeback_rate_detail don't have any currency, we will change it to a currency by default (Dollars)
+      chargeback_rate_detail = chargeback_rate_detail_stub.create(:chargeback_rate_detail_currency_id => nil)
+      migrate
+
+      expect(chargeback_rate_detail.reload.chargeback_rate_detail_currency_id).not_to be_nil
+    end
+
+    it "doesnt changes nil" do
+      # if the chargeback_rate_detail have a currency, its value must be the same after migration
```

Comment:
Same as above, move some of this comment into the `it` and this comment is unneeded.